HorticraftPricer.ahk

```diff
@@ -469,10 +469,10 @@ valuedictionary[RAW_Jewel_Implicit_Cluster] := Jewel_Implicit_Cluster
         }
 
         ; Append our found price, which will be "-" if no match was found
-        outputText := outputText "/" outPrice
+		outputText := outputText outPrice "/"
     }
-
-    clipboard := outputText
+	
+    clipboard := RegExReplace( outputText, "\/$" )
```

Comment:
We can use `SubStr(outputText, 1, -1)` which will be a fair bit faster than a regex.

Reply:
sounds good